### ISSUE TITLE
Any chance we can get a push in the right direction for using this in front of node?

### ISSUE THREAD

**Issue by samuelgoldenbaum** [heroku/heroku-buildpack-nginx#22]
Any chance we can get a push in the right direction for using this in front of node?

**Comment by beanieboi** [comment]
what is preventing you to start using it?

**Comment by samuelgoldenbaum** [comment]
@beanieboi  I have an existing test nodejs server. I added the buildpack as per docs. Followed the guide, but getting:
`2018-11-30T20:22:19.891556+00:00 app[web.1]: buildpack=nginx at=app-initialization
2018-11-30T20:22:20.010364+00:00 heroku[web.1]: Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
`

Repo is here for review if anyone can help: [https://github.com/samuelgoldenbaum/ngnixnode](https://github.com/samuelgoldenbaum/ngnixnode) I assue i have something incorrect in the `nginx.conf.erb` file.
I set the upstream to 127.0.0.1:8181 as per the port nodejs server is listening on.
